backend/clickhouse/migrations/000043_create_traces_sampling_table.up.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS traces_sampling (
+    Timestamp DateTime64(9),
+    UUID UUID,
+    TraceId String,
+    SpanId String,
+    ParentSpanId String,
+    ProjectId UInt32,
+    SecureSessionId String,
+    TraceState String,
+    SpanName LowCardinality(String),
+    SpanKind LowCardinality(String),
+    Duration Int64,
+    ServiceName LowCardinality(String),
+    ServiceVersion String,
+    TraceAttributes Map(LowCardinality(String), String),
+    StatusCode LowCardinality(String),
+    StatusMessage String,
+    Events Nested (
+        Timestamp DateTime64(9),
+        Name LowCardinality(String),
+        Attributes Map(LowCardinality(String), String)
+    ),
+    Links Nested (
+        TraceId String,
+        SpanId String,
+        TraceState String,
+        Attributes Map(LowCardinality(String), String)
+    )
+) ENGINE = MergeTree
+ORDER BY (
+        ProjectId,
+        toStartOfHour(Timestamp),
+        cityHash64(UUID)
+    ) SAMPLE BY cityHash64(UUID) TTL toDateTime(Timestamp) + toIntervalDay(30);
```

Comment:
should we be doing this with `SpanName` from the start?

Reply:
I was thinking some more about using `SpanName` as the primary key for the main `traces` table e.g. `ORDER BY (ProjectId, SpanName, Timestamp, UUID)`, but unless we're always filtering by `SpanName`, it will end up slowing things down